### PR TITLE
Alias directive v6

### PR DESCRIPTION
**Pull request by angrykoala**
# Description

This PR add tests for `@alias` directive with the new API.

The test is failing due to numericResolver not yet being supported

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
# Performance Report

No Performance Changes


<details>
    <summary>Show Full Table</summary>
    
| name | dbHits | old dbHits | time (ms) | old time (ms) | maxRows |
| ---- | ------ | ---------- | --------- | ------------- | ------- |
|  aggregations.TopLevelAggregate | 3404 | 3404 | 33 | 45 | 1134 |
|  aggregations.TopLevelAggregateWithMultipleFields | 6802 | 6802 | 43 | 55 | 1134 |
|  aggregations.NestedAggregation | 16544 | 16544 | 71 | 91 | 2174 |
|  aggregations.AggregationWithWhere | 11970 | 11970 | 40 | 45 | 2174 |
|  aggregations.AggregationWhereWithinNestedRelationships | 22116478 | 22116478 | 2194 | 2288 | 2008534 |
|  aggregations.AggregationWhereWithinNestedConnections | 22116478 | 22116478 | 2268 | 2216 | 2008534 |
|  aggregations.NestedCountFromMovieToActors | 9701 | 9701 | 32 | 35 | 2174 |
|  aggregations.NestedCountFromActorsToMovie | 9928 | 9928 | 34 | 42 | 2174 |
|  aggregations.DeeplyNestedCount | 11062184 | 11062184 | 2796 | 2410 | 2008534 |
|  aggregations.InterfacesAggregations | 6242 | 6242 | 50 | 48 | 2080 |
|  aggregations.InterfacesAggregationsWithTwoFields | 11444 | 11444 | 107 | 83 | 2080 |
|  batch-create.BatchCreate | 4200 | 4200 | 111 | 150 | 600 |
|  batch-create.BatchCreateSmall | 77 | 77 | 51 | 57 | 11 |
|  connect.createAndConnect | 14424 | 14424 | 170 | 170 | 3003 |
|  connections.Connection | 14049 | 14049 | 56 | 66 | 2174 |
|  connections.NestedConnection | 41068 | 41068 | 134 | 131 | 4516 |
|  create.SimpleMutation | 7 | 7 | 47 | 54 | 1 |
|  cypher-directive.TopLevelMutationDirective | 1135 | 1135 | 31 | 30 | 1134 |
|  delete.SimpleDelete | 19401 | 19401 | 212 | 195 | 1040 |
|  delete.NestedDeleteInUpdate | 19165 | 19165 | 125 | 150 | 1183 |
|  2871.NestedRelationshipFilter | 23073 | 23049 | 64 | 107 | 4429 |
|  2925.SingleRelationshipFilter | 6468 | 6468 | 35 | 40 | 1040 |
|  2925.NestedSingleRelationshipFilter | 22891 | 22891 | 66 | 102 | 2174 |
|  2925.SingleRelationshipRequiredFilter | 5201 | 5201 | 34 | 38 | 1040 |
|  2925.NestedSingleRelationshipRequiredFilter | 9361 | 9361 | 50 | 58 | 1040 |
|  query.SimpleQuery | 3121 | 3121 | 18 | 26 | 1040 |
|  query.SimpleQueryWithRelationship | 16129 | 16129 | 36 | 47 | 2174 |
|  query.QueryWhere | 9675 | 9679 | 29 | 43 | 2161 |
|  query.SimpleQueryWithNestedWhere | 9842 | 9846 | 42 | 51 | 2161 |
|  query.Nested | 10095502 | 10095502 | 6873 | 6697 | 2008534 |
|  query.NestedWithFilter | 10074351 | 10074351 | 6808 | 6468 | 2004000 |
|  query.OrFilterOnRelationships | 42627 | 42696 | 149 | 159 | 1754 |
|  query.OrFilterOnRelationshipsAndNested | 36302 | 37027 | 195 | 206 | 1754 |
|  query.QueryWithNestedIn | 13454 | 13218 | 46 | 56 | 1245 |
|  query.NestedConnectionWhere | 9801 | 9801 | 47 | 68 | 2174 |
|  query.DeeplyNestedConnectionWhere | 9850 | 9909 | 94 | 81 | 2174 |
|  query.DeeplyNestedWithRelationshipFilters | 18758 | 18848 | 110 | 139 | 1586 |
|  query.NestedWithRelationshipSingleFilters | 3865 | 3865 | 160 | 183 | 1134 |
|  query.Fulltext | 64 | 64 | 29 | 33 | 16 |
|  query.FulltextWithNestedQuery | 558 | 558 | 40 | 43 | 84 |
|  sorting-and-cypher.TopLevelSortWithCypher | 14059 | 14059 | 37 | 46 | 2174 |
|  sorting-and-cypher.TopLevelConnectionSortWithCypher | 14059 | 14059 | 60 | 62 | 2174 |
|  sorting-and-cypher.TopLevelSortWithCypherWithNested | 14204 | 14204 | 57 | 56 | 2174 |
|  sorting-and-cypher.TopLevelConnectionSortWithCypherWithNested | 14204 | 14204 | 87 | 93 | 2174 |
|  sorting-and-cypher.TopLevelSortWithExpensiveCypher | 14905 | 14926 | 124 | 115 | 2174 |
|  sorting-and-cypher.TopLevelConnectionSortWithExpensiveCypher | 14404 | 14404 | 112 | 140 | 2174 |
|  sorting.SortMultipleTypes | 3498 | 3498 | 77 | 91 | 1040 |
|  sorting.SortMultipleTypesWithCypherWithCypher | 14504 | 14504 | 94 | 111 | 2174 |
|  sorting.SortOnNestedFields | 14049 | 14049 | 40 | 47 | 2174 |
|  sorting.SortDeeplyNestedFields | 43148 | 43148 | 72 | 95 | 4516 |
|  sorting.ConnectionWithSort | 3281 | 3281 | 69 | 71 | 1040 |
|  unions.SimpleUnionQuery | 321 | 321 | 64 | 55 | 35 |
|  unions.SimpleUnionQueryWithMissingFields | 293 | 293 | 49 | 62 | 35 |
|  unions.NestedUnion | 410637 | 410637 | 267 | 262 | 33033 |
|  unions.NestedUnionWithMissingFields | 384611 | 384611 | 269 | 274 | 33033 |
|  update.NestedUpdate | 16143 | 16143 | 90 | 108 | 2002 |

</details>
        
Old Schema Generation: 27.205s
Schema Generation: 27.389s
Old Subgraph Schema Generation: 31.144s
Subgraph Schema Generation: 31.632s

File: packages/graphql/tests/api-v6/integration/directives/alias/query.int.test.ts [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    const testHelper = new TestHelper({ v6Api: true });
+
+    let Movie: UniqueType;
+    let Director: UniqueType;
+
+    beforeEach(async () => {
+        Movie = testHelper.createUniqueType("Movie");
+        Director = testHelper.createUniqueType("Director");
+
+        const typeDefs = `
```
Comment (MacondoExpress) [inline packages/graphql/tests/api-v6/integration/directives/alias/query.int.test.ts:33]:
```suggestion
        const typeDefs = /* GraphQL */ `
```

**Review by MacondoExpress — CHANGES_REQUESTED** [review]
Can you add a TCK test for it?

File: packages/graphql/tests/api-v6/tck/directives/alias/query.test.ts [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+                            }
+                        }
+                    }
+                }
+            }
+        `;
+
+        const result = await translateQuery(neoSchema, query, { v6Api: true });
+
+        // NOTE: Order of these subqueries have been reversed after refactor
```
Comment (MacondoExpress) [inline packages/graphql/tests/api-v6/tck/directives/alias/query.test.ts:70]:
```suggestion
```

**Review by MacondoExpress — APPROVED** [review]

**Pull request merged**